applications/dashboard/controllers/api/RolesApiController.php

```diff
@@ -243,6 +387,39 @@ public function post(array $body) {
     }
 
     /**
+     * Rename a legacy Vanilla permission slug.
+     *
+     * @param string $permission
+     * @return string
+     */
+    private function renamePermission($permission) {
+        $result = $permission;
```

Comment:
That line does nothing.

Reply:
This was done to help debug this method, now and in the future. It's easier to compare the original value with the result.

Reply:
I don't think we should let 'debug' code lie around. We can easily add that line back if we need to actually debug the thing. Removing it will also shut up the editors about unused variable.